src/components/StudyPane/Toolbar/Buttons.tsx

```diff
@@ -427,19 +428,83 @@ function setNumIndentById(chapters: any[], id: number, numIndent: number) {
 }
 
 export const NewStropheBtn = () => {
+  const { ctxNumSelectedWords, ctxNewStropheEvent, ctxSetNewStropheEvent} = useContext(FormatContext);
+
+  const [visibleOptions, setVisibleOptions] = useState(false);
+
+  const enabledButton = ()=>{
+    if(ctxNumSelectedWords===1){
+      return true;
+    }
+    else{
+      return false;
+    }
+  }
+
+  const buttonEnabled = enabledButton();
+
+  const handleClick = () => {
+    if(buttonEnabled){
+      ctxSetNewStropheEvent(true);
+      return;
+    }
+    else{
+      return;
+    }
+  }
 
   return (
+    <>
+    <div className="relative">
     <div className="flex flex-col group relative inline-block items-center justify-center px-2 xsm:flex-row">
       <button
         className="hover:text-primary"
-        onClick={() => console.log("New Strophe Clicked")} >
-        <CgArrowsBreakeV opacity="0.4" fontSize="1.5em" />
+        onClick={handleClick} >
+        <CgArrowsBreakeV opacity={(buttonEnabled)?`1`:`0.4`} fontSize="1.5em" />
+        <ToolTip text="New strophe" />
       </button>
-      <ToolTip text="New strophe" />
     </div>
+    </div>
+    </>
   );
 };
 
+export const MergeStropheBtn = () => {
+  const { ctxNumSelectedWords, ctxSetMergeStropheEvent } = useContext(FormatContext);
+
+  const enabledButton = ()=>{
+    if(ctxNumSelectedWords===1){
+      return true;
+    }
+    else{
+      return false;
+    }
+  }
+
+  const buttonEnabled = enabledButton();
+
+  const handleClick = () => {
+    if(buttonEnabled){
+      ctxSetMergeStropheEvent(true);
+      return;
+    }
+    else{
+      return;
+    }
+  }
+
+  return (
+    <div className="flex flex-col gropu relative inline-block items-center justify-center px-2 xsm:flex-row">
+      <button
+        className="hover:text-primary"
+        onClick={handleClick}>
+        <CgArrowsMergeAltV opacity={(buttonEnabled)?`1`:`0.4`} fontSize="1.5em" />
+      </button>
+      <ToolTip text="Merge strophe" />
```

Comment:
ToolTip should be wrapped inside the button component. Currently, this tooltip is not showing